README.md

```diff
@@ -45,4 +51,4 @@ We welcome contributions to this repository.
 See [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
 
 # License
-[MIT](LICENSE)
+[MIT](LICENSES/MIT.txt
```

Comment:
```suggestion
[MIT](LICENSES/MIT.txt)
```
**and**
I would recommend to keep the license file in top level and create a symlink `LICENSES/MIT.txt -> LICENSE.md`. I think that is also done in opossum: https://github.com/opossum-tool/OpossumUI/tree/main/LICENSES

Reply:
The CI only succeeds for a hard symlink, but fails fails for a soft one.
Are we ok with using a hard symlink?